.github/workflows/tag.yml

```diff
@@ -3,13 +3,13 @@ name: Application Release
 on:
   push:
     tags:
-      - "v*"
+      - "v1.*"
```

Comment:
Do we need this change? If we ever make a v2 we would still need a similar CI workflow, no?

Reply:
We need to distinguish "v1" tag from version tags (like v1.20.0) here to make sure that we don't trigger a release when we push "v1" tag to its new position. I changed it to "v*.*", wdyt ?

Reply:
Oh right, good catch! "v*.*" sounds good. Can you add a comment about this?

Reply:
Yes ! :smile: